quickstarts/datazoom/config.yml

```diff
@@ -0,0 +1,46 @@
+# Name of the quickstart (required)
+name: datazoom-quickstart
+
+# Displayed in the UI (required)
+title: Datazoom Quickstart
```

Comment:
```suggestion
title: Datazoom 
```

Reply:
Thanks for the contribution!  I don't think we need to call it a quickstart in the title as it's implicitly a quickstart.